clients/index.html

```diff
@@ -163,7 +163,7 @@
       <div class="col-md-10">
         <h3>Crowdmix</h3>
         <ul class="list-inline">
-          <li><i class="fa fa-globe color-green"></i> <a class="linked" href="http://crowdmix.me">crowdmix.me</a></li>
+          <li><i class="fa fa-globe color-green"></i> <a class="linked" href="http://www.crowdmix.me">crowdmix.me</a></li>
```

Comment:
Would you like to make this pull request separate from that discussion ? I.e. removal of the link would be a separate PR ?